test/services/user/services/AdminUsers.test.js

```diff
@@ -169,17 +171,19 @@ describe('AdminUsersService', () => {
 			account: {
 				userId: teacher._id,
 			},
-			$sort: sortObject,
+			query: {
+				$sort: sortObject,
+			},
 		});
 
 		const resultSortedByFirstName = await adminStudentsService.find(createParams({ firstName: -1 }));
 		expect(resultSortedByFirstName.data.lenght > 1);
 		expect(resultSortedByFirstName.data[0].firstName > resultSortedByFirstName.data[1].firstName);
 
 		const resultSortedByClass = await adminStudentsService.find(createParams({ class: -1 }));
-		expect(resultSortedByClass.data[0].class > resultSortedByClass.data[1].class);
-
+		expect(resultSortedByClass.data[0].classes[0] > resultSortedByClass.data[1].classes[0]);
 
+		/* TODO: Do not work!
```

Comment:
WIP

Reply:
No this test do not work all the time, it send a wrong query and get wrong answer and say all is ok. I do not know how i can test this case. Only if i rewrite all tests by adding own school with new students. But it is increase this PR a lot.